docs/micro-rdk/motor/gpio.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Configure a gpio motor"
+linkTitle: "gpio"
+weight: 10
+type: "docs"
+description: "Configure brushed or brushless DC motors."
+images: ["/icons/components/motor.svg"]
+# SMEs: Rand, James
+---
+
+The `gpio` model supports [DC motors](https://en.wikipedia.org/wiki/DC_motor) (both brushed and brushless).
+
+You can configure [encoders](/micro-rdk/encoder/) to work with `gpio` motors.
+Find more information in the [encoded motor documentation](/components/motor/gpio/encoded-motor/).
+
+To configure a DC motor as a component of your robot, first configure the [board](/components/board/) to which the motor driver is wired.
```

Comment:
Since this page is esp32-specific, should this sentence and/or the link in it be more specific?

Reply:
See comment below-- I think ok for now as micro-RDK will hopefully not always be just ESP32

Reply:
I see what you mean and thought about it more--shouldn't the link still go to micro-rdk/board?

Reply:
Yes, absolutely! Sorry for the confusion